ocaml/sdk-gen/component-test/README.md

```diff
@@ -0,0 +1,20 @@
+## How Component Testing Works
+
+#### jsonrpc-server
+
+jsonrpc-server is a HTTP server designed to mock XAPI. This server executes the method specified in the request and returns a response containing the result. It is important to note that the server reads test data from JSON files in the spec/ directory. Each JSON file should contain a JSON object with the fields of test_id, method, params, and expect_result.
```

Comment:
It's better to describe the idea on how to organize the test cases (specs).
At the beginning, it will be fine to trivially put all test cases in one directory. In the future, it would be nice if the test cases could be organized under sub-directories and with corresponding test IDs.

Reply:
Updated.